src/bot/events/trackMemberBans.js

```diff
@@ -0,0 +1,33 @@
+import {EventListener} from 'yuuko';
+import config from '../../../config';
+import {formatDateRelative} from '../util/discord';
+
+export default new EventListener('guildBanAdd', (guild, member, {db, client}) => {
+	setTimeout(async () => {
```

Comment:
What's the purpose of waiting 5 seconds before handling the event?

Reply:
Just in case it takes a while to create the record we query for. We ban before creating the record right? If it's too long I can reduce it

Reply:
Eh, yeah, I guess that works for me